source/includes/options-mongodrdl-commands.yaml

```diff
@@ -0,0 +1,265 @@
+program: mongodrdl
+name: sample
+directive: commandoption
+description: |
+  Samples the namespaces specified by the
+  :option:`--db <mongodrdl --db>`, optional
+  :option:`--collection <mongodrdl --collection>`, and other
+  :ref:`command line options <mongodrdl-command-line-options>` to
+  output a :ref:`.drdl <drdl>` file. The ``sample`` command
+  encapsulates all of :binary:`~bin.mongodrdl` behavior prior to version
+  2.11.
+
+  .. code-block:: sh
+
+     mongodrdl sample --db <db-name> --collection <collection-name> --out <filename>.drdl
+---
+program: mongodrdl
+name: list-schema-ids
+directive: commandoption
+description: |
+  Lists the :ref:`ObjectId <document-bson-type-object-id>` of each
+  stored schema and the date it was created. 
+
+  ``list-schema-ids`` requires the following option:
+  
+  .. list-table::
+     :header-rows: 1
+     :widths: 25 75
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
```

Comment:
Note to self for copy review edits: Link these to the option in the main list lower on the page.